Dockerfile

```diff
@@ -1,12 +1,23 @@
 FROM openjdk:11-jre-slim
 
+LABEL org.opencontainers.image.authors="Fraunhofer AISEC <codyze@aisec.fraunhofer.de>" \
+      org.opencontainers.image.licenses="Apache-2.0" \
+      org.opencontainers.image.source="https://github.com/Fraunhofer-AISEC/codyze" \
+      org.opencontainers.image.vendor="Fraunhofer AISEC" \
+      org.opencontainers.image.base.name="openjdk:11-jre-slim"
```

Comment:
Does it make sense to have this as a label? Seems redundant

Reply:
While we host on GitHub, this information is readily available. In this sense it's redundant. Do we intend to distribute Codyze container images somewhere else?

Reply:
> While we host on GitHub, this information is readily available. In this sense it's redundant. Do we intend to distribute Codyze container images somewhere else?

For now, its only planned to host it on GH.

Reply:
Then I'll remove it.